src/patternfly/utilities/Spacing/examples/spacing-combined-example.hbs

```diff
@@ -1,4 +1,4 @@
-{{#> title titleType="h2" pf-c-title--Modifier="pf-m-lg pf-m-margin"}}
+{{#> title titleType="h2" title--modifier="pf-m-lg pf-m-margin"}}
```

Comment:
pf-m-margin can be removed in the titles component for all the spacing examples too.

Reply:
fixed